### ISSUE TITLE
Some code in the guide is still Python 2

### ISSUE THREAD

**Issue by fjhheras** [LxMLS/lxmls-guide#143]
For example, Python 3 would fail with the current `print` statements and `xrange` in exercises 3.9 and 3.11.

**Comment by ramon-astudillo** [comment]
thanks! can you do a PR?

**Comment by fjhheras** [comment]
Yes, I can. Do you want it to be merged before the beginning of the course? Otherwise I can wait in case I see other instances of the problem elsewhere in the guide and solve all of them at once
